src/test/java/uk/ac/cam/cl/dtg/segue/auth/SegueLocalAuthenticatorTest.java

```diff
@@ -267,4 +263,40 @@ public final void segueLocalAuthenticator_setPasswordAndImmediateAuthenticate_co
 			fail("We expect a user to be returned");
 		} 
 	}
+
+	@Test
+	public void ensureValidPassword_validString() {
+		SegueLocalAuthenticator segueAuthenticator = new SegueLocalAuthenticator(this.userDataManager, this.passwordDataManager,
+				this.propertiesLoader, possibleAlgorithms, preferredAlgorithm);
+		try {
+			segueAuthenticator.ensureValidPassword("Password123!");
+		} catch (InvalidPasswordException e) {
+			fail("We expect this password to be accepted as valid");
```

Comment:
I know you have just reused the `fail` that was already imported, but you may as well import the one from the right namespace. The same goes for `assertEquals`. It's just a matter of replacing the namespaces for both of them.